torch/testing/_internal/common_cuda.py

```diff
@@ -32,3 +35,43 @@ def initialize_cuda_context_rng():
         for i in range(torch.cuda.device_count()):
             torch.randn(1, device="cuda:{}".format(i))
         __cuda_ctx_rng_initialized = True
+
+
+def tf32_is_not_fp32():
+    if not torch.cuda.is_available():
+        return False
+    if torch.cuda.get_device_properties(torch.cuda.current_device()).major < 8:
```

Comment:
you don't check cuda version here? E.g. if someone compiles pytorch with cuda 10 and runs it on ampere, there's no tf32 yet your check will return True

Reply:
fixed